src/js/util/util.js

```diff
@@ -0,0 +1,15 @@
+import { MAX_CAR_NAME_LENGTH, RANDOM_NUM } from './constatns.js';
+
+export function toNameArray(str) {
+	return str.split(',').map((item) => item.trim());
+}
+
+export function isCheckCarNameLength(carName) {
+	return carName.every((item) => item.length <= MAX_CAR_NAME_LENGTH);
+}
+
+export function isMoveCar() {
+	const randomNum =
+		Math.floor(Math.random() * RANDOM_NUM.MAX_NUM) + RANDOM_NUM.MIN_NUM;
+	return randomNum >= RANDOM_NUM.FORWARD_NUM;
```

Comment:
```suggestion
	const { FORWARD_NUM, MIN_NUM, MAX_NUM } = RANDOM_NUM
	const randomNum = Math.floor(Math.random() * MAX_NUM) + MIN_NUM;
	return randomNum >= FORWARD_NUM;
```
RANDOM_NUM이 많이 사용되니 이렇게 바꿔봐도 좋을거같네요 👀 
코드를 보다가 느낀건데 코드 포메팅이 전혀 안되고있어서 prettier를 적용해보시는걸 추천드릴게요~
정의한 설정파일에 따라 자동으로 파일 포메팅을 변경해줘서 통일감있게 코드를 유지보수할 수 있어요

Reply:
헛! prettier를 에디터내에서 사용중인데 포매팅이 안돼서 보이는걸까요?? 🥺

Reply:
아 에디터 내에서 따로 사용중이셨군요!
별도로 설정 파일이 없길래 사용안하시는 줄 알았어요~
제가 수정다가 포메팅이 안맞아서 코멘트 드렸었어요 👀

Reply:
앗 `.prettierrc` 파일을 같이 커밋하긴 했었는데 혹시 추가로 설정해줘야 하는 파일이 있을까요?? 👀👀👀

Reply:
엇 아뇨 해당 파일만 있으면 됩니다~ 제 세팅에 뭔가 문제가 있었던거 같네요.. 🤔